src/app/rundown-view/components/rundown-header-context-menu/rundown-header-context-menu.component.ts

```diff
@@ -48,8 +52,38 @@ export class RundownHeaderContextMenuComponent {
     },
   ]
 
+  private readonly contextMenuOptionsForRehearseRundown: ContextMenuOption[] = [
+    {
+      label: 'Activate (On Air)',
+      contextAction: (): void => this.openActivateRundownDialog(),
+    },
+    {
+      label: 'Deactivate',
+      contextAction: (): void => this.openDeactivateRundownDialog(),
+    },
+    {
+      label: 'Take',
+      contextAction: (): void => this.take(),
+    },
+    {
+      label: 'Reset Rundown',
+      contextAction: (): void => this.openResetRundownDialog(),
+    },
+    {
+      label: 'Reingest data',
+      contextAction: (): void => this.reingestData(),
+    },
+  ]
```

Comment:
Instead of duplicating the same entries as in the Active menu options, you can just add them to this array:
```
private readonly contextMenuOptionsForRehearseRundown: ContextMenuOption[] = [
    {
      label: 'Activate (On Air)',
      contextAction: (): void => this.openActivateRundownDialog(),
    },
    ...this.contextMenuOptionsForActiveRundown,
  ]
```

Reply:
Done